after/syntax/perl.vim

```diff
@@ -0,0 +1,38 @@
+if dracula#should_abort('perl')
+    finish
+endif
+
+" Regex
+hi! link perlMatchStartEnd       DraculaRed
+
+" Builtin functions
+hi! link perlOperator            DraculaCyan
+hi! link perlStatementFiledesc   DraculaCyan
```

Comment:
Is there perhaps a parent scope here that hits all of these (e.g. `perlStatement`)?

Would be better to just target that rather than each individually.

Reply:
I tried looking, but unfortunately perlStatement also covers control flow e.g `for`, `while` and phasers e.g `BEGIN`, `INIT` so I had to pick them out individually.